web/api/v2/create-action/index.ts

```diff
@@ -0,0 +1,153 @@
+import { errorResponse } from "@/api/helpers/errors";
+import { getAPIKeyGraphqlClient } from "@/api/helpers/graphql";
+import { verifyHashedSecret } from "@/api/helpers/utils";
+import { validateRequestSchema } from "@/api/helpers/validate-request-schema";
+import { generateExternalNullifier } from "@/lib/hashing";
+import { ApolloError } from "@apollo/client";
+import { NextRequest, NextResponse } from "next/server";
+import * as yup from "yup";
+import {
+  CreateDynamicActionMutation,
+  getSdk as createDynamicActionSdk,
+} from "./graphql/create-dynamic-action.generated";
+import { getSdk as fetchApiKeySdk } from "./graphql/fetch-api-key.generated";
+
+const createActionBodySchema = yup.object({
+  app_id: yup.string().strict().required(),
+  action: yup.string().strict().required(),
+  team_id: yup.string().strict().required(),
```

Comment:
@andy-t-wang I think it will be more consistent to get this from the URL because we are doing it like this in other endpoints. But in this case, I think that this endpoint is more related to the app. Thus URL should look like this: `/api/v2/team/[team_id]/app/[app_id]/create-action`. But it will be too long. 

Or we can just leave it as it is and get this data from the body. 

What do you think

Reply:
no lets get it from the body